### PR TITLE
Improve readability of section trees

### PR DESCRIPTION
**Pull request by poornimanayar**
Noticed that Section Trees uses a mixture of _italics_ and `code` to mention code.  I have made it uniform

### CODE REVIEW

**Comment by poornimanayar** [comment]
Referring to issue https://github.com/umbraco/UmbracoDocs/issues/4416

**Review by sofietoft — COMMENTED** [review]
Thanks for the PR @poornimanayar !

I see you ran into a lot of long sentences as well 😅 Would you be up for resolving those at well, by any chance?

Also, on a side note:
We'll soon be migrating the UmbracoDocs to a new platform, and in that process we will be leaving docs for 6, 7 and 8 behind on Our. So, that means you're free to skip these versions if you plan on doing more cleanup 💪 😄

**Comment by poornimanayar** [comment]
Ah @sofietoft I can certainly help with the guidance of Vale linting.

**Comment by sofietoft** [comment]
Sounds good! 💪

**Comment by poornimanayar** [comment]
@sofietoft I think I have addressed the comments as much as I can. Any more ideas on how to resolve some of the other comments?

File: Extending/Section-Trees/Trees/index-v8.md [hunk 1]
```diff
@@ -132,9 +134,9 @@ protected override TreeNode CreateRootNode(FormDataCollection queryStrings)
...
 
-For example clicking on one of the 'Favourite Things' in the custom tree example outlined above will trigger the loading of an 'edit.html' view from the folder: */views/favouriteThingsAlias/edit.html*. The 'Delete' menu item would also load a view from: */views/favouriteThingsAlias/delete.html*
+For example clicking on one of the 'Favourite Things' in the custom tree example outlined above will trigger the loading of an 'edit.html' view from the folder: `/views/favouriteThingsAlias/edit.html`. The 'Delete' menu item would also load a view from: `/views/favouriteThingsAlias/delete.html`
 
-If you're creating a custom tree as part of an Umbraco package/plugin, it's recommended to change the location of the default folder to the `App_Plugins` folder. You achieve this by decorating you mvc *TreeController* with the *PluginController* attribute.
+If you're creating a custom tree as part of an Umbraco package/plugin, it's recommended to change the location of the default folder to the `App_Plugins` folder. You achieve this by decorating you mvc `TreeController` with the `PluginController` attribute.
```
Comment (sofietoft) [inline Extending/Section-Trees/Trees/index-v8.md:139]:
```suggestion
When creating a custom tree as part of a Umbraco package, it is recommended to change the location of the default folder. It should be changed to the `App_Plugins` folder. You achieve this by decorating your MVC `TreeController` with the `PluginController` attribute.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for giving this a shot @poornimanayar ! 💪 

I've made some suggestions to resolve the remaining sentences - would you double-check to make sure I didn't loose the meaning of the sentences while doing it? 😁 🤞

**Comment by poornimanayar** [comment]
All done @sofietoft

**Comment by sofietoft** [comment]
Thanks @poornimanayar !
Just took care of the final vale error.
I think we're all good now 💪

**Pull request merged**